lib/review_bot/notification.rb

```diff
@@ -9,18 +9,14 @@ def initialize(pull_request:, suggested_reviewers:)
     attr_reader :pull_request, :suggested_reviewers
 
     def message
-      [needs_review_from, pull_request.html_url].join("\n")
-    end
-
-    private
-
-    def needs_review_from
       [
-        %("#{pull_request.title}" needs a #{needed_review_type} from),
+        %(• <#{pull_request.html_url}|#{pull_request.title}> needs a *#{needed_review_type}* from),
```

Comment:
Me likey! I think the PR number would be helpful in there still. Maybe at the end?

change(notification): simplify (?) the formatting of the reminder #27 like github does?

change(notification): simplify (?) the formatting of the reminder [#27] for more differentiation?